docker/install.sh

```diff
@@ -107,7 +107,7 @@ elif [[ $build_ros2_pkgs -eq 1 ]]; then
     cd ~/workspace_ros2
     source /opt/ros/humble/setup.bash
 
-    colcon build --packages-up-to ssc_interface_wrapper_ros2 --build-base ./build_ssc_interface_wrapper --install-base /opt/carma/install --cmake-args -DCMAKE_BUILD_TYPE=Release
+    colcon build --packages-up-to ssc_interface_wrapper_ros2 driver_shutdown_ros2 --build-base ./build_ssc_interface_wrapper --install-base /opt/carma/install --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
Seems like ros1 and ros2 packages are being installed in the same directory, can we move ros2 install to install_ros2

Reply:
It is okay to do this because for noetic and humble, it is completely different ubuntu. Therefore, either image will only have `/opt/carma/install`